src/utilities/utilities.js

```diff
@@ -169,3 +170,44 @@ export const handleSelfFocus = e => {
     e.currentTarget.focus();
   }
 };
+
+export const isFullyVisible = (slideIndex, config) => {
```

Comment:
I also removed `positionValue` from the passed in arguments since I realized I wasn't even using it :scream: